packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -445,10 +452,13 @@ class _SnackBarState extends State<SnackBar> {
             ),
           ),
           if (widget.action != null)
-            ButtonTheme(
-              textTheme: ButtonTextTheme.accent,
-              minWidth: 64.0,
```

Comment:
Is the minWidth already accounted for in the TextButton.styleFrom()?

Reply:
Yes, the default minimum size for TextButton is Size(64, 36)